app/models/ingredient.rb

```diff
@@ -0,0 +1,6 @@
+class Ingredient < ApplicationRecord
+  has_one_attached :image
```

Comment:
Add validations on image
size not more than 5 mb
allow only jpg/jpeg or png

Reply:
Done